Testing/Functional/SmokeTest/UpdateSelenium.ps1

```diff
@@ -60,6 +60,42 @@ function Confirm-NeedForUpdate{
     return ([System.Version]$v2).Major -lt ([System.Version]$v1).Major
 }
 
+function DownLoadDriver{
+    param(
+        [Parameter(Mandatory=$true)]
+        [Uri]
+        $DownloadUrl,
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $WebDriversPath,
+        [Parameter(Mandatory=$true)]
+        [ValidateScript({Test-Path -Path $_ -PathType Container})]
+        [string]
+        $DriverTempPath
+    )
+
+    Write-Debug -Message "Dowloading $DownloadUrl"
+
+    try {
+        Invoke-WebRequest $DownloadUrl -OutFile "$DriverTempPath\chromeNewDriver.zip" 2>&1 | Out-Null
```

Comment:
I'm not completely sure if this is necessary or not, but you might consider adding `-ErrorAction "Stop"` to this line. We've been burned in the past by PowerShell's default error action being `Continue` (which would cause the `catch` to never be triggered). `Invoke-WebRequest` might be ok (I tested it and the `catch` did trigger in my test), but in other places where we use it, we do specify the error action, so it might be good to do it here, if nothing else for consistency.

For example:
https://github.com/cisagov/ScubaGear/blob/25613115a018511430f2674c411512b9d8ca468c/PowerShell/ScubaGear/Modules/Providers/ExportEXOProvider.psm1#L122

Reply:
In this case I am not looking for it t Stop on Error.  I want the try catch block to handle it.

Reply:
That's the thing, try/catch doesn't catch non-terminating errors. Unless a command "stops", try/catch won't handle it.
- https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_try_catch_finally?view=powershell-7.4
- https://stackoverflow.com/questions/41476550/try-catch-not-working-in-powershell-script
- https://stackoverflow.com/questions/1142211/try-catch-does-not-seem-to-have-an-effect

Reply:
Okay-  let me re-work it.

Reply:
Fixed issues